armi/utils/plotting.py

```diff
@@ -1296,11 +1295,10 @@ def _makeComponentPatch(component, position, cold):
                 patch = matplotlib.patches.Polygon(corners, fill=True)
                 blockPatch.append(patch)
         else:
-            # Just make it a hexagon...
+            # Just make it a hexagon
             blockPatch = matplotlib.patches.RegularPolygon(
                 (x, y), 6, radius=component.getDimension("op", cold=cold) / math.sqrt(3)
```

Comment:
I think this one needs to be rotated as well.

Reply:
done